.github/workflows/on-release.yaml

```diff
@@ -23,6 +23,8 @@ jobs:
 #            args: '--target x86_64-apple-darwin'
           - platform: 'windows-latest'
             args: ''
+          - platform: 'ubuntu-24.04'
+            args: ''
```

Comment:
Rather than compiling on 24.04, it would be better to compile on older versions (an Ubuntu 20.04 would work well) to avoid compatibility issues on older systems. I'm thinking particularly about glibc.

Reply:
I'm more into supporting the latest LTS that is not EOL. So at the moment, this is 20.04 until April 2025.